controller/src/main/java/io/pravega/controller/server/security/auth/handler/impl/AclAuthorizerImpl.java

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.controller.server.security.auth.handler.impl;
+
+import io.pravega.auth.AuthHandler;
+import lombok.NonNull;
+
+class AclAuthorizerImpl extends AclAuthorizer {
```

Comment:
Should we move this class to a common module like "auth" such that schema registry and other services can leverage the auth module

Reply:
As discussed, I've created a separate issue #5118 for the reorganization of the security-related code in general.